Jenkinsfile

```diff
@@ -39,7 +39,7 @@ pipeline {
                 maven cmd: 'clean verify ' +
                       '-Dmaven.test.failure.ignore=true ' +
                       "-DdeployApplicationName=msgraph-connector-${deployApplicationName} " +
-                      "-Dengine.page.url=${params.engineSource} " +
+                      "-Dengine.list.url=${params.engineSource} " +
```

Comment:
why list instead of page? did we change that in core?

Reply:
engine.page.url does not exist in my opinion?! never have worked...

Reply:
I think the plugin was always responsive to `engine.list.url` . see http://axonivy.github.io/project-build-plugin/release/9.4/installEngine-mojo.html#engineListPageUrl

However, maybe we copied from the wrong-build pipeline. There are a few RCPTT builds, that determine the engine/designer to use by other means (ant-scripts). Therefore they work with a different parameter ....